Source/Diagnostics/ComputeDiagFunctors/ParticleReductionFunctor.cpp

```diff
@@ -81,7 +90,9 @@ ParticleReductionFunctor::operator() (amrex::MultiFab& mf_dst, const int dcomp,
                 amrex::ParticleReal ux = p.rdata(PIdx::ux) / PhysConst::c;
                 amrex::ParticleReal uy = p.rdata(PIdx::uy) / PhysConst::c;
                 amrex::ParticleReal uz = p.rdata(PIdx::uz) / PhysConst::c;
-                amrex::Real value = map_fn(xw, yw, zw, ux, uy, uz);
+                amrex::Real value;
+                if ((do_filter) && (filter_fn(xw, yw, zw, ux, uy, uz) <= 0.5)) value = 0._rt;
```

Comment:
If #3007 is merged, this condition will need to be changed for consistency, as will the one on line 133.

Reply:
Ok, merged :) Ready for update :)